torch/csrc/jit/ir/ir.h

```diff
@@ -72,6 +72,11 @@ using namespace ::c10::attr;
 namespace aten {
 using namespace ::c10::aten;
 }
+namespace cuda {
+#ifdef __HIP_PLATFORM_HCC__
```

Comment:
You want to hide it from RoCM, aren't you?
```suggestion
#ifndef __HIP_PLATFORM_HCC__
```